<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMutualismBreakdownerParametersWidget</class>
 <widget class="QWidget" name="QtMutualismBreakdownerParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>764</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="button_save">
     <property name="text">
      <string>&amp;Save parameters</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="button_set_random_values">
     <property name="text">
      <string>Set &amp;random values</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="box_n_timesteps">
     <property name="maximum">
      <number>1000000000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="box_delta_t">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_n_timesteps">
     <property name="text">
      <string>#timesteps</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="tab_initial_condistions">
      <attribute name="title">
       <string>Initial conditions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QLabel" name="label_initial_loripes_density">
         <property name="text">
          <string>Initial loripes density (# m^-2)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="box_initial_loripes_density">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_initial_organic_matter_density">
         <property name="text">
          <string>Initial organic matter density</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="box_initial_organic_matter_density">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>10000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_init_seagrass_density">
         <property name="text">
          <string>Initial seagrass density (# m^-2)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="box_initial_seagrass_density">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_initial_sulfide_concentration">
         <property name="text">
          <string>Initial sulfide concentration</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="box_initial_sulfide_concentration">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_detoxification">
      <attribute name="title">
       <string>Detoxification</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_detoxifaction_max_rate">
         <property name="text">
          <string>Detoxifaction max rate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="box_detoxification_max_rate">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_detoxifaction_min">
         <property name="text">
          <string>Detoxifaction min</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="box_detoxification_min">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_detoxifaction_rate">
         <property name="text">
          <string>Detoxifaction rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="box_detoxification_rate">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_consumption">
      <attribute name="title">
       <string>Loripes consumption</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="box_consumption_by_loripes">
         <property name="title">
          <string>Loripes sulfide consumption</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Mutualism breakdown</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_mutualism_breakdown_max">
         <property name="text">
          <string>Mutualism breakdown max</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="box_mutualism_breakdown_max">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_mutualism_breakdown_r0">
         <property name="text">
          <string>Mutualism breakdown r0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="box_mutualism_breakdown_r0">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_mutualism_breakdown_rate">
         <property name="text">
          <string>Mutualism breakdown rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="box_mutualism_breakdown_rate">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_organic_matter">
      <attribute name="title">
       <string>Organic matter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label_organic_matter_addition">
         <property name="text">
          <string>Organic matter addition</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="box_organic_matter_addition">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_organic_matter_breakdown">
         <property name="text">
          <string>Organic matter breakdown</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="box_organic_matter_breakdown">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_organic_matter_capture">
         <property name="text">
          <string>Organic matter capture</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="box_organic_matter_capture">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_organic_matter_to_sulfide_factor">
         <property name="text">
          <string>Organic matter to sulfide factor</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="box_organic_matter_to_sulfide_factor">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_organic_matter_to_sulfide_rate">
         <property name="text">
          <string>Organic matter to sulfide rate</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="box_organic_matter_to_sulfide_rate">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_poisoning">
      <attribute name="title">
       <string>Poisoning</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="box_poisoning">
         <property name="title">
          <string>Poisoning</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_recruitment">
      <attribute name="title">
       <string>Recruitment</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_recruitment_max">
         <property name="text">
          <string>Recruitment max</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="box_recruitment_max">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_recruitment_rate">
         <property name="text">
          <string>Recruitment rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="box_recruitment_rate">
         <property name="decimals">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_seagrass">
      <attribute name="title">
       <string>Seagrass</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QLabel" name="label_seagrass_carrying_capacity">
         <property name="text">
          <string>Seagrass carrying capacity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="box_seagrass_carrying_capacity">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_seagrass_growth_rate">
         <property name="text">
          <string>Seagrass growth rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="box_seagrass_growth_rate">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_seagrass_to_organic_matter_factor">
         <property name="text">
          <string>Seagrass to organic matter factor</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="box_seagrass_to_organic_matter_factor">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_sulfide">
      <attribute name="title">
       <string>Sulfide</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QLabel" name="label_sulfide_diffusion_rate">
         <property name="text">
          <string>Sulfide diffusion rate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="box_sulfide_diffusion_rate">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>100000000000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_delta_t">
     <property name="text">
      <string>Delta t</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="button_load">
     <property name="text">
      <string>&amp;Load parameters</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_set_random_values</tabstop>
  <tabstop>button_load</tabstop>
  <tabstop>button_save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
